zenoh-jni/src/config.rs

```diff
@@ -119,6 +120,81 @@ pub extern "C" fn Java_io_zenoh_jni_JNIConfig_00024Companion_loadYamlConfigViaJN
     })
 }
 
+/// Obtains the id of the config, returning it as a little endian byte array.
+#[no_mangle]
+#[allow(non_snake_case)]
+pub unsafe extern "C" fn Java_io_zenoh_jni_JNIConfig_00024Companion_getIdViaJNI(
```

Comment:
`Config::id` is not part of the API, it's only available with `unstable`.

Reply:
Ok, removed it. For some confusion I thought it was available